bluetooth_mesh/test/test_scene_client.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=W0621, C0103
```

Comment:
There's an inconsistency as to where tests are stored. Some are in `bluetooth_mesh/tests`, some are stored next to modules they tests. I propose the schema we use in our repos: package `tests` on the repo level, which contains `unit` & `functional` packages - that enables easy launching of a specific level of tests.

Reply:
Ok, but not now.

Reply:
Understandable. Should I create an issue so that it doesn't get lost?

Reply:
https://silvair.atlassian.net/browse/SG-393